src/sagemaker/workflow/steps.py

```diff
@@ -294,16 +313,18 @@ def arguments(self) -> RequestType:
         NOTE: The `CreateTrainingJob` request is not quite the args list that workflow needs.
         The `TrainingJobName` and `ExperimentConfig` attributes cannot be included.
         """
+        if self.run_args:
+            request_dict = self.run_args
+        else:
+            self.estimator._prepare_for_training()
+            train_args = _TrainingJob._get_train_args(
+                self.estimator, self.inputs, experiment_config=dict()
```

Comment:
Right now, we don't support experiment_config in any job steps. We need to massage the run_args in the new approach as well.

Reply:
good catch, need to pop them out. However, IIRC, although given, the backend will just ignore it, right? need to call this out in the doc.